src/components/markdown-resource-selector.js

```diff
@@ -0,0 +1,69 @@
+import { Tooltip } from 'antd';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+import { LinkOutlined } from '@ant-design/icons';
+import React, { Fragment, useState } from 'react';
+import ResourceSelectorDialog from './resource-selector/resource-selector-dialog.js';
+
+function MarkdownResourceSelector({ size, disabled, onUrlSelect }) {
```

Comment:
Since we don't expose an enum-like variable with the possible options for size, maybe we can simply have a `small` prop which is `boolean`. I did something similar in `empty-state` with a `compact` prop.

Reply:
Good idea, done! (also for `markdown-help.js`)